html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-generated-content.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<title>fieldset generated content</title>
```

Comment:
Maybe link to bugs / spec PR / add your name as the author?

Reply:
Naw, that is in the git log, and spec link is implied by the dir structure.